pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -57,12 +57,12 @@ class Build {
     Integer getJavaVersionNumber() {
         // version should be something like "jdk8u" or "jdk" for HEAD
         def matcher = (buildConfig.JAVA_TO_BUILD =~ /(\d+)/)
-        if (matcher.matches()) {
+        if (matcher.find()) {
             List<String> list = matcher[0] as List
             return Integer.parseInt(list[1] as String)
         } else if ("jdk".equalsIgnoreCase(buildConfig.JAVA_TO_BUILD.trim())) {
             // This needs to get updated when JDK HEAD version updates
```

Comment:
With your update to use `buildConfig.JAVA_FEATURE_VERSION` we should remove that comment as it's no longer a hard-coded value.

Reply:
On second thought. Will `JAVA_FEATURE_VERSION` be available at this point. Looks to me that there are only these parameters going to be generated:
https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/build/common/create_job_from_template.groovy#L64

Reply:
I'm not too sure, I didn't realise only this list of values we're passed into the groovy scripts. Surely just adding JAVA_FEATURE_VERSION to this list would be a good idea as we don't want too many hardcoded values that we'd need to change in 6 months time

Reply:
Or adding it as a constant somehow?